examples/pipe/dev/system.clj

```diff
@@ -30,20 +45,56 @@
                                 (filter #(re-find re (:topic-name %))))]
       (ja/delete-topics! client topics-to-delete))))
 
+
+(defn start-producer []
+  (jc/producer producer-config))
+
+(defn stop-producer [producer]
+  (.close producer))
+
+(defn start-consumer []
+  (-> (consumer-config)
+      (jc/consumer)
+      (jc/subscribe [(pipe/topic-config "output")])
+      (jc/seek-to-beginning-eager)))
+
+(defn stop-consumer [consumer]
+  (.close consumer))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;; Utility functions ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(defn list-topics []
```

Comment:
The README kept referring to the `list-topics`, `publish` and `get-keyvals` functions which were not defined in the project, so I defined them here. I replaced `get-keyvals` with `consume`.